src/containers/WeatherList.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Chart from '../components/Chart';
+import Table from 'react-bootstrap/lib/Table';
+
+class WeatherList extends Component {
+    renderWeather(cityWeatherData) {
+        const name = cityWeatherData.city.name;
+        const temps = cityWeatherData.list.map(weather => weather.main.temp);
+        const humidity = cityWeatherData.list.map(weather => weather.main.humidity);
+        const pressure = cityWeatherData.list.map(weather => weather.main.pressure);
+        return (
+            <tr key={name}>
+                <td>{name}</td>
+                <td>
+                    <Chart data={temps} color='orange' units='K'/>
+                </td>
+                <td>
+                    <Chart data={humidity} color='blue' units='hPa'/>
+                </td>
+                <td>
+                    <Chart data={pressure} color='green' units='%'/>
+                </td>
+            </tr>
+        );
+    }
+
+    render() {
+        return (
+            <Table>
```

Comment:
nit: `Table` -> `table`

Reply:
`Table` is being imported (`import Table from 'react-bootstrap/lib/Table'`) and used thanks to the `react-bootstrap` utility.